### DISCUSSION TITLE
Is INFERNO supposed to have pistol start?

### DISCUSSION THREAD

**Discussion by klearscreen** [Owlet7/wadfusion#30] [Q&A] [answered]
The master levels from Willits to Titan had pistol start on all levels, but the level pack for INFERNO carries weapons and ammo between levels. Is that intended?

**Comment by Owlet7** [comment, marked as answer]
The logic that I went by there is that most of the maps were intended as single, unconnected levels, so I think they should force pistol starts.

It's mentioned in the readme of one of the Titan maps (`trouble.wad`) that "you're being sent again, but as usual, you
only get a pistol", so despite it being treated as a series of maps in Fusion, they also have pistol starts because of this.

The Cabal maps were updated to work as a connected episode, so they don't force pistol starts.

The Inferno levels were also originally released as separate maps, but since the story never mentions it (unlike Titan), I thought that it would work better to not force a pistol start on them. I dunno, should I revert that decision?

**Reply by klearscreen** [reply]
Interesting, I was under the impression that having a pistol start was a requirement for master levels.
If INFERNO and CABAL were not planned for it, I don't think it should be forced.

**Reply by klearscreen** [reply]
The level "The Image of Evil" from the Cabal series has a pistol start, seems like an oversight.

**Reply by Owlet7** [reply]
It was intentional, I thought that because the previous level ends on a damaging floor, it would make sense to reset to a pistol start. I've since remembered that there's an exit behind a wall, which can be used to avoid taking too much damage, so I removed the pistol start from the current build, and it won't be there in the next version.
